mkt/api/paginator.py

```diff
@@ -39,13 +39,13 @@ def page(self, number):
         number = self.validate_number(number)
         bottom = (number - 1) * self.per_page
         top = bottom + self.per_page
-        page = Page(self.object_list[bottom:top], number, self)
 
         # Force the search to evaluate and then attach the count. We want to
         # avoid an extra useless query even if there are no results, so we
         # directly fetch the count from hits.
         # Overwrite `object_list` with the list of ES results.
```

Comment:
That comment is no longer true, needs updating - we are not directly overwriting. I am pretty surprised it still works without needing the overwrite, btw. Can you double check that we are not making extra ES queries ? (I don't fully trust our tests here)